nvflare/app_common/workflows/cross_site_model_eval.py

```diff
@@ -128,7 +128,9 @@ def start_controller(self, fl_ctx: FLContext):
 
         # If the list of participating clients is not provided, include all clients currently available.
         if not self._participating_clients:
-            self._participating_clients = [c.name for c in engine.get_clients()]
+            engine.sync_clients_from_main_process()
```

Comment:
Instead of asking user to call this,
Can we put `sync_clients_from_main_process()` inside the `get_clients()` method of the engine?

Reply:
No, we need to keep both methods. The behavior is different when calling from parent or child process.